test/fixtures/addresses.ts

```diff
@@ -4,6 +4,8 @@ export default {
   USDC: '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8',
   USDT: '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9',
   GAUGE: '0x97E2768e8E73511cA874545DC5Ff8067eB19B787',
```

Comment:
Should this be old gauge, just for someone without context?

```suggestion
  OLD_GAUGE: '0x97E2768e8E73511cA874545DC5Ff8067eB19B787',
```